repo-agent/Makefile

```diff
@@ -19,6 +19,18 @@ ifndef GITHUB_PAT
 $(error GITHUB_PAT is not set. Please set it before running make.)
 endif
 
+ifndef ANTHROPIC_API_KEY
```

Comment:
We need to update the installer.sh to take in these envvars. 
And update the readme to show MT vs ST installation.,